File: backend/src/main/java/com/kh/backend/domain/alarm/model/vo/Alarm.java
package com.kh.backend.domain.alarm.model.vo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Alarm {
	
	private int alarmNo;
	private int userNo;
	private int fromUserNo;
	private String content;
	private char status; // "N - 안읽음 Y - 읽음"
	private String alarmDate; // 일단 String
	private char typeCode; // B, C, L, M, R, P
	private int refNo;
	
	private Type type;

}

File: service-frontend/src/type/alarm.ts
export interface alarm {
    alarmNo : number,
    userNo : number,
    fromUserNo : number,
    content : string,
    status : string,
    alaramDate : string,
    typeCode : string,
    refNo : number,

    fromUserNickName : string
}

export const initialAlarm: alarm = {
    alarmNo : 0,
    userNo : 0,
    fromUserNo : 0,
    content : '',
    status : '',
    alaramDate : '',
    typeCode : '',
    refNo : 0,
    fromUserNickName : ''
}

export const initialAlarmList: alarm[] = [];